src/www/ui/page/AdminLicenseCandidate.php

```diff
@@ -118,7 +118,12 @@ protected function handle(Request $request)
       case 'verify':
       case 'variant':
         $rfParent = ($request->get('do')=='verify') ? $rf : $suggest;
-        $ok = $this->verifyCandidate($rf,$shortname,$rfParent);
+        try{
+          $ok = $this->verifyCandidate($rf,$shortname,$rfParent);
+        } catch (\Throwable $th) {
+          $vars['message'] = 'The license text already exists.';
+          break;
+        }      
```

Comment:
was just going thru your build logs and found an extra space here, please remove it, this should fix the build error.

Reply:
Corrected.